lib/osf-components/addon/components/resource/component.ts

```diff
@@ -0,0 +1,71 @@
+import { inject as service } from '@ember/service';
+import Component from '@glimmer/component';
+import Intl from 'ember-intl/services/intl';
+
+import ResourceModel from 'ember-osf-web/models/resource';
+
+interface ResourceArgs {
+    resource: ResourceModel;
+}
+
+export default class ResourceComponent extends Component<ResourceArgs> {
```

Comment:
That sounds good and I agree. They're not quite final though since they can be removed. Would something along the lines of `ObtainedBadges` or `CompletedBadges` work? It is a little more descriptive, while also adding positive incentive for earning them.

Reply:
This component and its import/export statements have now been updated to OpenResource. The name is a hybridization of the parent div's name in the mock-ups here:

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/82047646/183147997-b0326b76-b8fd-41eb-8f4a-772ed608cf42.png">

while still being somewhat succinct. It also trends away from the Badges naming convention as recommended and takes into consideration Brian's earlier conversation regarding finalized. Please let me know if you like it, and can make any requested changes.